### DISCUSSION TITLE
Does your plugin supports android12-qpr1-release?

### DISCUSSION THREAD

**Discussion by holybora** [pvoid/AndroidBlueprintPlugin#21] [Q&A]
When I try to add AOSP sources to IDEA it says me next: 
![Screenshot from 2022-02-09 10-49-09](https://user-images.githubusercontent.com/86054220/153160011-aa3946fc-ca08-425b-9cd3-75fb3aa9400d.png)

How to resolve it?  

env:
 **Ubuntu 20.04 
 IDEA 2021.3
 AOSP branch: android12-qpr1-release
 lunch sdk_car_x86_64-userdebug
 Selected Android SDK 32 
 Build has been successful.**
 
 FYI 
 I have tired open project already by **development/tools/idegen** and it was almost successful. May be this is reason of error? For now IDEA can't find some androidx dependencies.

**Comment by pvoid** [comment]
This message is not from the plugin, so I don't really know what's about.
